### PR TITLE
Created Landing Pages using HTML and CSS

### PR DESCRIPTION
**Pull request by kumari-arya**
### Landing Page
This is a website created using HTML and CSS.The website link is attached below-

Landing_Page

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+                    <p class="first-header-subtext">This website has some subtext that goes under the main title. It's a smaller font and the color is lower contrast</p>
+                    <button class="first-button">Sign up</button>
+                </div>
+                <div class="placeholder">
+                <p >this is a placeholder for an image</p>
+                </div>
+            </div>
+        </div>
+        <div class="second">
+            <h1 class="second-header">Some random information</h1>
```
Comment (roopaljasnani) [inline index.html]: An HTML page cannot have multiple h1 tags

File: index.html [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+        <div class="top">
+            <div class="header">
+                <p class="logo">Header Logo</p>
+                <div class="links">
+                    <a href="#">header link one</a>
+                    <a href="#">header link two</a>
+                    <a href="#">header link three</a>
+                </div>
+            </div>
+            <div class="first">
```
Comment (roopaljasnani) [inline index.html]: consider using section tags for emphasizing blocks

File: index.html [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+                    <h1 class="first-header">This website is awesome</h1>
+                    <p class="first-header-subtext">This website has some subtext that goes under the main title. It's a smaller font and the color is lower contrast</p>
+                    <button class="first-button">Sign up</button>
+                </div>
+                <div class="placeholder">
+                <p >this is a placeholder for an image</p>
+                </div>
+            </div>
+        </div>
+        <div class="second">
```
Comment (roopaljasnani) [inline index.html]: consider using section tags for emphasizing blocks

File: index.html [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+                    <p>this is some subtext under an illustration or image</p>
+                </div>
+                <div class="image-subtext">
+                    <img src="images/image1.jpg">
+                    <p>this is some subtext under an illustration or image</p>
+                </div>
+            </div>
+        </div>
+
+        <div class="third">
```
Comment (roopaljasnani) [inline index.html]: consider using section tags for emphasizing blocks,

for this particular section - a figure & a blockquote tag can also be considered

File: index.html [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+            <p class="quote">This is an inspiring quote, or a testimonial from a customer. Maybe it's just filling up space, or maybe people will actually read it. Who knows? All I know is that it looks nice.</p>
+            <p class="quoter">-Thor, God of Thunder</p>
+        </div>
+
+        <div class="fourth">
+            <div class="fourth-text">
+                <p  class="fourth-header">Call to action! It's time!</p>
+                <p class="fourth-subtext">Sign up for our product by clicking that button right over there!</p>
+            </div>
+            <button class="fourth-button"> Sign-up</button>
```
Comment (roopaljasnani) [inline index.html]: remove space before button

File: index.html [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+
+        </div>
+        <div class="footer">
```
Comment (roopaljasnani) [inline index.html]: use footer  tag here

File: style.css [hunk 7]
```diff
@@ -0,0 +1,194 @@
+
```
Comment (roopaljasnani) [inline style.css]: remove space from here

File: style.css [hunk 8]
```diff
@@ -0,0 +1,194 @@
...
+body,
+.top,
+.second,
+.third,
+.image-subtext,
+.fourth-text{
+    display : flex;
+    flex-direction: column;
+
```
Comment (roopaljasnani) [inline style.css]: remove space

File: style.css [hunk 9]
```diff
@@ -0,0 +1,194 @@
...
+.first,
+.images-second,
+.fourth,
+.links{
+    display: flex;
+}
+
+.second-header{
+    text-align: center;
+}
```
Comment (roopaljasnani) [inline style.css:21]: add space after each block

File: style.css [hunk 10]
```diff
@@ -0,0 +1,194 @@
...
+.image-subtext p{
+    width:180px;
+    text-align: center;
+}
+
+.images-second
+{
```
Comment (roopaljasnani) [inline style.css]: formatting in the line is entirely off

File: style.css [hunk 11]
```diff
@@ -0,0 +1,194 @@
...
+    margin-left: auto;
+    margin-right: auto;
```
Comment (roopaljasnani) [inline style.css]: use margin shorthand

File: style.css
Comment (roopaljasnani) [inline style.css:1]: I think I've repeated this point now for all your PRs - Pls make a habit of working hand-in-hand with helpful tools like [ESLint](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint) & [Prettier](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode) that come along with the VSCode editor & configure them to run & format files automatically on save.

I should not see the need to give this reminder again please, @kumari-arya.

**Review by roopaljasnani — CHANGES_REQUESTED** [review]
Another point that I've continually highlighted for your PRs is - 

It is important that you file out other meta details in your PR like a proper title, description (with proper screenshots/screen recordings when some UI work is involved), assignee, reviewers, etc.

Read about how to properly fill in such details in this [guide from our handbook](https://github.com/jalantechnologies/handbook/blob/main/engineering/pr-etiquette.md).

Pls also follow [this PR template](https://github.com/jalantechnologies/boilerplate-next-typescript/blob/develop/.github/pull_request_template.md) when filling out the description in this & upcoming PRs.

Pls follow this by default and should not need reminding again, @kumari-arya

**Comment by kumari-arya** [comment]
> Another point that I've continually highlighted for your PRs is -
> 
> It is important that you file out other meta details in your PR like a proper title, description (with proper screenshots/screen recordings when some UI work is involved), assignee, reviewers, etc.
> 
> Read about how to properly fill in such details in this [guide from our handbook](https://github.com/jalantechnologies/handbook/blob/main/engineering/pr-etiquette.md).
> 
> Pls also follow [this PR template](https://github.com/jalantechnologies/boilerplate-next-typescript/blob/develop/.github/pull_request_template.md) when filling out the description in this & upcoming PRs.
> 
> Pls follow this by default and should not need reminding again, @kumari-arya

Updated. Please see once again

**Comment by kumari-arya** [comment]
Resolved all raised issues.

**Review by roopaljasnani — APPROVED** [review]

**Pull request merged**